.github/workflows/testsuite.yml

```diff
@@ -31,4 +29,16 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
-      - run: cd tests && meteor npm install && meteor npm run test
```

Comment:
Why?

Reply:
Just a normalisation compared to other packages: e.g. redis-oplog. I restore the previous one.